yolact.py

```diff
@@ -470,6 +481,16 @@ def __init__(self):
         self.detect = Detect(cfg.num_classes, bkg_label=0, top_k=cfg.nms_top_k,
             conf_thresh=cfg.nms_conf_thresh, nms_thresh=cfg.nms_thresh)
 
+
+        # set default backbone weights
+        self.init_weights(backbone_path='weights/' + cfg.backbone.path)
+
+        # GPU
+        #TODO try half: net = net.half()
```

Comment:
Off topic, but has anyone tried running the models as half/fp16? Generally improves performance,mem footprint.